Source/SwiftLintBuiltInRules/Rules/Idiomatic/ExplicitInitRule.swift

```diff
@@ -201,7 +201,7 @@ private extension ExplicitInitRule {
             }
 
             correctionPositions.append(violationPosition)
-            let newNode = node.with(\.calledExpression, calledBase.trimmed)
+            let newNode = node.with(\.calledExpression, calledBase.with(\.trailingTrivia, []))
```

Comment:
Can also be just that:
```suggestion
            let newNode = node.with(\.calledExpression, calledBase)
```
But yours is fine as well. 👍